### PR TITLE
스토리보드 디자인 적용 ver.1

### PR DESCRIPTION
**Pull request by ZZANZU**

### CODE REVIEW

File: app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeProductViewHolder.kt [hunk 1]
```diff
@@ -11,9 +11,10 @@ import kotlinx.android.synthetic.main.item_customer_home_list.view.*
  * Created by user on 2018-05-22.
  */
 class CustomerHomeProductViewHolder(itemView: View?) : RecyclerView.ViewHolder(itemView) {
-    var companyImage : ImageView = itemView!!.findViewById(R.id.customer_home_product_img)
-    var companyName : TextView = itemView!!.findViewById(R.id.customer_home_product_company)
-    var productPriceBefore : TextView = itemView!!.findViewById(R.id.customer_home_product_cost_before)
-    var productPriceAfter : TextView = itemView!!.findViewById(R.id.customer_home_product_cost_after)
-    var productSurplus : TextView = itemView!!.findViewById(R.id.customer_home_product_surplus)
+    var companyImage: ImageView = itemView!!.findViewById(R.id.customer_home_product_img)
```
Comment (kujyp) [inline app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeProductViewHolder.kt]: https://kotlinlang.org/docs/tutorials/android-plugin.html#view-binding

CustomerHomeProductViewHolder.kt
```
class CustomerHomeProductViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    fun bind(item: CustomerHomeProduct) {
        Log.d()

        itemView.customer_home_product_img.setImageResource(item.companyImage)
        itemView.customer_home_product_company.text = item.companyName
        itemView.customer_home_product_name.text = item.productName
        itemView.customer_home_product_cost_before.text = item.productPriceBefore
        itemView.customer_home_product_cost_after.text = item.productPriceAfter
        itemView.customer_home_product_surplus.text = item.productSurplus
    }
}
```

CustomerHomeProductAdapter.kt
```
override fun onBindViewHolder(holder: CustomerHomeProductViewHolder, position: Int) {
    Log.d()
    holder.bind(productItems[position])
}
```

File: app/src/main/res/drawable/rounded_corner.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="#FFFFFF"/>
+    <solid android:color="@color/white"/>
```
Comment (kujyp) [inline app/src/main/res/drawable/rounded_corner.xml]: 이것도 가능함 `@android:color/white`

**Review by kujyp — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeSearchFragment.kt [hunk 3]
```diff
@@ -5,20 +5,37 @@ import android.support.v4.app.Fragment
...
  */
-class CustomerHomeSearchFragment: Fragment() {
+class CustomerHomeSearchFragment: Fragment(), OnMapReadyCallback {
+    private lateinit var mMap: GoogleMap
+
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         Log.d("customer home search fragment created!")
         val mView = inflater.inflate(R.layout.fragment_customer_home_search, container, false)
 
+        val mapFragment = mView.findViewById(R.id.customer_search_map) as MapView
```
Comment (kujyp) [inline app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeSearchFragment.kt:24]: synthetic property사용가능

File: app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeProductViewHolder.kt (hunk 1 above)
Comment (ZZANZU) [reply app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeProductViewHolder.kt]: wow

File: app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeSearchFragment.kt (hunk 3 above)
Comment (ZZANZU) [reply app/src/main/java/io/github/tturiyo/tturiyo_android/ui/customer/CustomerHomeSearchFragment.kt:24]: val mapFragment = mView.customer_search_map as MapView
이렇게?

**Review by kujyp — APPROVED** [review]

**Pull request merged**